ui/src/pages/backoffice/Document/DocumentDetails/DocumentDetails.js

```diff
@@ -30,15 +32,24 @@ class DocumentHeader extends Component {
       <>
         <label>Document</label> #{data.metadata.pid}{' '}
         <CopyButton text={data.metadata.pid} />
+        {(
+          <>
+            <br />
+            <label>Created by</label>{' '}
+          </>
+        ) && data.metadata.created_by}
         <br />
-        <label>Created by</label> {data.metadata.created_by}
-        <br />
-        <label>On</label> {toShortDate(data.created)}
+        <label>Created on</label> {toShortDate(data.created)}
       </>
     );
     return (
       <DetailsHeader
-        title={data.metadata.title}
+        title={
+          <>
+            <Header.Subheader>{data.metadata.document_type}</Header.Subheader>
+            {data.metadata.title} - ed. {data.metadata.edition}
```

Comment:
Should you add also the imprint year here?

Reply:
I will in the next PR